style.css

```diff
@@ -152,3 +305,871 @@ ul#horizontal-list li {
   display: inline;
   padding: 5px;
 }
+
+.recentworks {
+  display: grid;
+  background-color: white;
+  align-self: center;
+  justify-content: center;
+  padding: 90px 0;
+  position: relative;
+  width: 375px;
+  left: 0;
+  top: 0;
+}
+
+.recent {
+  width: 327px;
+  height: 83px;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-self: center;
+  color: #1c1a19;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 0;
+  margin-left: 24px;
+}
+
+.vector {
+  width: 327px;
+  height: 0;
+  border: 0;
+  flex: 1;
+  padding: 0;
+  order: 0;
+  margin-bottom: 80px;
+  margin-left: 24px;
+  align-self: center;
+}
+
+.gymfit {
+  width: 327px;
+  height: 237px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding: 0;
+  margin: 0 24px;
+}
+
+.stories {
+  width: 327px;
+  height: 36px;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding: 0;
+  margin: 24px 24px 12px;
+}
+
+.passage {
+  width: 327px;
+  height: 147px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  padding: 0;
+  margin: 0 24px;
+}
+
+.tags {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  align-self: flex-start;
+  padding: 0;
+  margin: 12px 24px;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+  order: 0;
+  flex-grow: 0;
+}
+
+#show {
+  display: flex;
+  align-self: flex-start;
+  text-align: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  order: 0;
+  justify-items: left;
+  border: none;
+  width: 140px;
+  height: 48px;
+  margin: 0 24px 24px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+#show:enabled {
+  background-color: #f55800;
+}
+
+#show:hover {
+  background-color: #fff5e1;
+}
+
+#show:active {
+  background-color: #e05304;
+}
+
+#show:disabled {
+  background-color: #f4f1f0;
+}
+
+.bg {
+  display: flex;
+  order: 4;
+  padding-left: 24px;
+}
+
+.stories2 {
+  position: absolute;
+  top: 940px;
+  width: 295px;
+  height: 81px;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #fff;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  padding: 0;
+  padding-left: 12px;
+  margin: 24px;
+}
+
+.passage2 {
+  position: absolute;
+  top: 1055px;
```

Comment:
As asked by the first reviewer, it is mandatory to use Flexbox to lay out the elements. 

Here you use `position: absolute;` to lay out the element and you use `display: flex; ` to lay out the child of the element.
Please remove `position: absolute;` and use Flexbox instead.